src/WebJobs.Extensions.DurableTask/ActorScheduler/ActorRef.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// A reference to an actor, used to identify actor instances.
+    /// </summary>
+    public struct ActorRef : IEquatable<ActorRef>, IComparable
+    {
+        /// <summary>
+        /// Create an actor reference for an actor.
+        /// </summary>
+        /// <param name="actorClass">the name of the actor class.</param>
+        /// <param name="actorKey">the actor key.</param>
+        public ActorRef(string actorClass, string actorKey)
```

Comment:
Naming: One potential worry I have about the name `ActorRef` is that it gives the impression that it is a pointer to an actor, just like in C# how reference parameters are pointers to a variable defined elsewhere. I think we observed this same confusion when reviewing early iterations of this with the team (someone even thought it was a like a C# proxy type). In fact, I think akka uses [ActorRef](https://doc.akka.io/api/akka/2.0/akka/actor/ActorRef.html) as a proxy type as well.

Could we consider an alternate name, like `ActorId`? Service Fabric uses the term [ActorId](https://docs.microsoft.com/en-us/dotnet/api/microsoft.servicefabric.actors.actorid?view=azure-dotnet), as do other implementations, so there is some prior art here. I feel like the meaning will be generally more easily understood by developers.